modules/odc-deploying-java-applications.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// applications/application_life_cycle_management/odc-deploying-java-applications.adoc
+
+[id="odc-deploying-java-applications_{context}"]
+= Upload JAR files for easy deployment of Java applications
+
+You can use the JAR files in the *Topology* view of the *Developer* perspective to deploy your Java applications.
+
+The following procedure explains the *Upload JAR file* option in the *Developer* perspective to deploy a Java application.
+
+.Procedure
+
+. In the *Topology* view, click the *Add to Project* menu item, which appears when you right-click on the *Topology* surface to see the *Upload JAR file* option. You can alternatively drag and drop the JAR file on the *Topology* surface.
+. Click on the *Upload JAR file* option to open the *Upload JAR file* form.
+. In the *Upload JAR file* form:
+* In the *JAR file* field, browse the JAR file on your local machine and upload the same or you can drag-and-drop it on the field.
```

Comment:
a toast alert is displayed at the top right if incompatible file type is dragged and dropped on topology

Reply:
Fixed!